addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2930,28 +2982,23 @@ export class OdooEditor extends EventTarget {
                                     sel.collapseToEnd();
                                 }
                             },
-                        },
-                        {
-                            groupName: 'paste',
-                            title: 'Paste as text',
-                            description: 'Simple text paste.',
-                            fontawesome: 'fa-font',
-                            callback: () => {},
-                        },
-                    ];
-
-                    const execCommandAtStepIndex = (index, callback) => {
-                        this._historyRevertUntil(index);
-                        this.historyStep(true);
-                        this._historyStepsStates.set(peek(this._historySteps).id, 'consumed');
+                        });
+                    }
+                    baseEmbedCommand.push({
+                        groupName: 'paste',
+                        title: 'Paste as text',
+                        description: 'Simple text paste.',
+                        fontawesome: 'fa-font',
+                        callback: () => {},
+                    });
 
+                    const undoAndApplyCommand = (callback) => {
+                        this.historyUndo();
```

Comment:
`historyUndo` cannot work. Here is two examples: https://drive.google.com/file/d/1w8J4UNuEiF6edvsCoAWgZXNpWOcuyBw2/view